graphql/src/index.ts

```diff
@@ -120,16 +120,25 @@ const resolvers = {
       const connectionArguments = { before, after, first, last };
       const { from, size } = getEsOffsetPaginationQuery(connectionArguments);
 
-      if (isDefined(orderByDistance) && isDefined(orderByName)) {
+      const orderByArgs = {
+        orderByDistance,
+        orderByName,
+        orderByAccessibilityProfile,
+      };
+
+      const isOrderByAmbiguous =
+        Object.values(orderByArgs).map(isDefined).filter(Boolean).length > 1;
```

Comment:
Isn't the mapping here redundant? Cannot the isDefined be used with a filter directly?

Reply:
Taking this into account in PR #114